src/Reflection/ReflectionClass.php

```diff
@@ -69,28 +69,16 @@ class ReflectionClass implements Reflection
     /** @var ClassLikeNode */
     private $node;
 
-    /**
-     * @var ReflectionClassConstant[]|null indexed by name, when present
-     * @psalm-var ?array<string, ReflectionClassConstant>
-     */
+    /** @var array<string, ReflectionClassConstant>|null indexed by name, when present */
```

Comment:
these break IDE support, sadly 😩

Reply:
However there's a lot of `<>` syntax in master in `@var`, `@param` annotations :)

Reply:
Yeah, true that, but I'll probably end up adding `Foo[]` in multiple locations as I go through the code 😬

Reply:
I've just found https://plugins.jetbrains.com/plugin/12754-phpstan--psalm--generics

Reply:
Tried that few weeks ago: not sure if it works now 😅

Reply:
Ok, after some thought put into this: the folks at @jetbrains will never get moving if we aren't the ones pushing it.

I'm fine with sacrificing some auto-completion to push the boundaries of what's done at type-level within our own annotations :+1:

Reply:
The plugin works fine for several use cases. Though eg. support for iterables is not ready yet.

Folks at Jetbrains are not planning to add support for these annotations anytime soon AFAIK :/

Reply:
[WI-47158](https://youtrack.jetbrains.com/issue/WI-47158) has been the [most upvoted PHPStorm feature request ever](https://youtrack.jetbrains.com/issues?q=project:PhpStorm%20order%20by:%20votes) for a while and there's still no response from @JetBrains...